### PR TITLE
Updating ksw to 0.1.0a

### PR DESCRIPTION
**Pull request by nh13**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by kyleabeauchamp** [comment]
I imagine there's an easier way to do submodules?  I would hope at least...

File: recipes/ksw/build.sh [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
-#export CFLAGS="-I$PREFIX/include"
-#export LDFLAGS="-L$PREFIX/lib"
-#export CPATH=${PREFIX}/include
+set -euo pipefail
+
+# These are used ksw's makefile
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export CFLAGS="-I$PREFIX/include"
```
Comment (kyleabeauchamp) [inline recipes/ksw/build.sh]: minor nit: wouldn't you want to use the same style of escaping in all the exports in this block?

Comment (kyleabeauchamp) [reply recipes/ksw/build.sh]: e.g. {}

**Comment by nh13** [comment]
@kyleabeauchamp I think the issue is the tarball generated by github doesn't have the submodule inside, so really I either (1) I have to produce custom-tarballs for each release, or (2) figure out how to get github to do it.  After this PR, I'll be doing (1).

**Comment by kyleabeauchamp** [comment]
SGTM

**Pull request merged**